src/seedu/addressbook/Main.java

```diff
@@ -15,6 +11,10 @@
 import seedu.addressbook.storage.StorageFile.StorageOperationException;
 import seedu.addressbook.ui.TextUi;
 
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
```

Comment:
Why change the place of import statements? 
Later on, you will learn that there is a fixed standard you will have to follow for order of import statements, so avoid changing things when unrelated to PR